src/Algebra/Graph/Relation.hs

```diff
@@ -517,7 +517,7 @@ induce p (Relation d r) = Relation (Set.filter p d) (Set.filter pp r)
 -- compose x                'empty'            == 'empty'
 -- compose ('vertex' x)       y                == 'empty'
 -- compose x                ('vertex' y)       == 'empty'
--- compose x                (compose y z)    == compose (compose x y) z
+-- compose x                (compose y z)      == compose (compose x y) z
```

Comment:
This is: 1) unrelated to the PR, and 2) seems to break the alignment of examples. Please revert!

Reply:
Okay, I just happened to see that '==' sign and gave it a little space when checking for comments. I am going to revert it!

Reply:
Thanks! These spaces are all well thought through: they make sure the documentation is nice to read after running Haddock.